.gitignore

```diff
@@ -1,4 +1,5 @@
 create_database.py
 app/database.db
 .idea
-**/__pycache__
\ No newline at end of file
+**/__pycache__
+app/logs
```

Comment:
add data here too, you dont want to commit that

Reply:
done

Reply:
not done, but ok, you did that differently

Reply:
I removed data folder